AutoDAGsExternalModule.php

```diff
@@ -8,7 +8,8 @@ class AutoDAGsExternalModule extends \ExternalModules\AbstractExternalModule{
 	// REDCap::getGroupNames() doesn't pick up on added or renamed groups until the next request.
 	private $groupsByID;
 
```

Comment:
This hook accepts additional arguments ($survey_hash, $response_id, $repeat_instance). Thoughts on whether I should include them?

I also dug in a bit and figured out that this name `hook_save_record` is legacy behavior from an earlier version of the external modules class. REDCap still does the right thing with it, even though it doesn't match the current documentation.

Reply:
Let's go ahead and include them. I'm a fan of clearly documented methods, even though standard practice from REDCap has been to leave off additional arguments if they are not used. From an implementation perspective they don't really give us much as I doubt we would use them here, but if I was a developer looking at this in the future it would be good to know they exist even if they don't effect anything.